package.json

```diff
@@ -8,8 +8,7 @@
     "node": ">=0.12.0",
     "npm": ">=3.0.0"
   },
-  "main": "src/type.js",
-  "browser": "lib/type.js",
+  "main": "lib/type.js",
```

Comment:
Exactly, I hadn't noticed this, thanks for fixing!

There should be a "jsnext:main" entry pointing to the src version too though, since that's used in some build tools that prefer dealing with the es6 source (like rollup and gulp-metal). Can you add it? There's an example [here](https://github.com/metal/metal-jsx/blob/master/package.json#L11).